js/src/components/shipping-rate-section/minimum-order-card/minimum-order-input-control.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppInputPriceControl from '.~/components/app-input-price-control';
+import AppButtonModalTrigger from '.~/components/app-button-modal-trigger';
+import MinimumOrderInputLabelText from './minimum-order-input-label-text';
+import EditMinimumOrderModal from './edit-minimum-order-modal';
+import './minimum-order-input-control.scss';
+
+const MinimumOrderInputControl = ( props ) => {
+	const { countryOptions, value, onChange } = props;
+	const { countries, threshold, currency } = value;
+
+	const handleBlur = ( event, numberValue ) => {
+		if ( numberValue === value.threshold ) {
+			return;
+		}
+
+		onChange( {
+			countries,
+			threshold: numberValue === 0 ? undefined : numberValue,
```

Comment:
💅  or 🚧 
If `0` is an invalid value, I'd suggest moving this logic to `<MinimumOrderCard>` and fix the inconsistency logics at:
1. https://github.com/woocommerce/google-listings-and-ads/blob/82bd0bc22cd14718ec9b01de0d23a23e025ff73d/js/src/components/shipping-rate-section/minimum-order-card/add-minimum-order-modal.js#L30-L35
2. https://github.com/woocommerce/google-listings-and-ads/blob/82bd0bc22cd14718ec9b01de0d23a23e025ff73d/js/src/components/shipping-rate-section/minimum-order-card/edit-minimum-order-modal.js#L38-L43

Reply:
Fixed in [`af7f16a` (#1266)](https://github.com/woocommerce/google-listings-and-ads/pull/1266/commits/af7f16a8ec39c6f6998966d54f85a282d833a492).